packages/i18n/src/locales

```diff
@@ -0,0 +1 @@
+../../../apps/meteor/packages/rocketchat-i18n/i18n
```

Comment:
Any specific reason to use a symbolic link here? Since we are replacing TAPPi18n completely, wouldn't be better to remove the rocketchat-i18n from the meteor packages folder?

Reply:
the lingo hub must to be configured and to be honest I'm not fluent with that tool so I'm keeping this for next steps